lib/main.js

```diff
@@ -26,6 +26,10 @@ export default {
       type: "string",
       default: "-Wall -std=c++11"
     },
+    clangDefaultCpp14Flags: {
+      type: "string",
```

Comment:
@steelbrain The reason it's a string that gets split by Space, it would look weird to users to have to separate parameters by comma.